lib/pbench/test/functional/server/test_connect.py

```diff
@@ -18,8 +18,10 @@ def test_connect(self, server_client: PbenchServerClient):
         assert "uri" in endpoints
 
         # verify all the required authentication fields are present
-        auth_fields = ["issuer", "client", "realm"]
-        assert all(e in endpoints["authentication"].keys() for e in auth_fields)
+        assert all(
+            e in endpoints["authentication"].keys()
+            for e in {"issuer", "client", "realm"}
+        )
```

Comment:
Given that this is now four lines instead of two, I liked the previous code better; however, I [still think](https://github.com/distributed-system-analysis/pbench/pull/3065#discussion_r1036311633) the "superset" operator ([`>=`](https://docs.python.org/3/library/stdtypes.html#frozenset.issubset)) is better.

Reply:
Fixed